mem-pool.c

```diff
@@ -44,16 +44,19 @@ void mem_pool_init(struct mem_pool *pool, size_t initial_size)
 	if (initial_size > 0)
 		mem_pool_alloc_block(pool, initial_size, NULL);
 
-	trace_printf_key(&trace_mem_pool, "mem_pool (%p): init (%"PRIuMAX") initial size\n",
-		pool, (uintmax_t)initial_size);
+	trace_printf_key(&trace_mem_pool,
```

Comment:
again, i'll leave this one for your series and not try to pull it into the FSMonitor series.